### ISSUE TITLE
Debug console: Horizontal bar partially out of screen & output of "Print" output wont fit

### ISSUE THREAD

**Issue by ItsCubeTime** [armory3d/armory#2190]
![image](https://user-images.githubusercontent.com/20190653/117696065-34593c80-b1c1-11eb-8ee7-f0094f103cf1.png)

As you can see, theres no word wrapping resulting in the actual message being printed not ever showing in my case.

You can also see how the text at the very top is partially out of view, **this happens only after printing something to the console**.

Both issues mentioned persist when setting the debug console position to "center".

Recorded on version 2021.05.0 - Blender 2.83LTS

Windows 10

**Comment by ItsCubeTime** [comment]
> I wonder if it would be better to be able to resize the debug console or whether the position infos ("Sources/armory/logicnode/PrintNode.hx:13") should be omitted in the debug console. Position infos can be quite helpful to see where something was printed from but I guess that it would be ok to omit them here?

Yes knowing where prints come from is something I missed out on when working with UE4. Ive seen lots of topics online related to finding print messages that people have forgotten where its called from so I think this is a nice to have.

But it does clutter the view a bit at the same time, maybe it could be colored differently (I would give it a gray/darker color to make it take less attention as you probably know where its emitted from most of the time), or give it like a seperate section/column a little to the left?

**Comment by ItsCubeTime** [comment]
Also regarding resizing, I think either (or even better both) word wrapping or resizing would do.

**Comment by knowledgenude** [comment]
I guess is better to keep the path and split it in two lines in a branch style:


`Sources/etc. PrintNode
 Abc`

Plus, if the previous path is the same (most of the times will be like that), no need to repeat it all the times, so we got

`Sources/etc. PrintNode:
 Abc
 Efg`

**Comment by ItsCubeTime** [comment]
I mean you could always hack together word wrapping by counting the number of characters in the string and inserting new lines based on that

**Comment by onelsonic** [comment]
Hi guys,
I struggle with this before as well.
But took the problem the other wy around and went after the font used instead and changed it when debugging.
Try with this one : https://www.dafont.com/fr/pixelated.font?text=qwertyuiopasdfghjklzxcvbnm1234567890
rename it to **font_default.ttf**
for your project.
It worked for me, let me know if it does for you?

**Comment by onelsonic** [comment]
Hi guys,
I struggled with this before as well.
But took the problem the other way around and went after the font used instead and changed it when debugging.
Try with this one : https://www.dafont.com/fr/pixelated.font?text=qwertyuiopasdfghjklzxcvbnm1234567890
rename it to **font_default.ttf**
for your project.

Also what about making the file location where the trace function is called as an option : so it just print out the variables?

**Comment by MoritzBrueckner** [comment]
> EDIT: Also what about making the file location where the trace function is called as an option : so it just print out the variables?

That sounds like a good idea :)

**Comment by onelsonic** [comment]
Is there any quick and dirty way to make the debug console goes FULLSCREEN ?
I played with this part here but no changes to the debug console...
https://github.com/armory3d/armory/blob/caa03b59c5be3ac6bbcdef922b75879eea35f067/Sources/armory/trait/internal/DebugConsole.hx#L184

**Comment by onelsonic** [comment]
I found that you can also make the debug console goes Fullscreen by changing this value here : 
https://github.com/armory3d/armory/blob/caa03b59c5be3ac6bbcdef922b75879eea35f067/Sources/armory/trait/internal/DebugConsole.hx#L182
